packages/liquid-html-parser/src/stage-1-cst.ts

```diff
@@ -105,6 +107,21 @@ export interface ConcreteBasicNode<T> {
   locEnd: number;
 }
 
+export interface ConcreteLiquidDocParamNode
+  extends ConcreteBasicNode<ConcreteNodeTypes.LiquidDocParamNode> {
+  name: string;
```

Comment:
Shouldn't this be `param` or `@param` as a const rather than an arbitrary string?